djautotask/management/commands/atsync.py

```diff
@@ -106,23 +108,28 @@ def handle(self, *args, **options):
             sync_classes.append(sync_classes.pop(0))
 
         for sync_class, obj_name in sync_classes:
+            error_msg = None
             try:
                 self.sync_by_class(sync_class, obj_name,
                                    full_option=full_option)
 
-            except AutotaskAPIException as e:
-                msg = 'Failed to sync {}. Autotask API returned an error: ' \
-                      '{}.'.format(obj_name, ' '.join(e.response.errors))
+            except AutotaskProcessException as e:
+                error_msg = ERROR_MESSAGE_TEMPLATE.format(
+                    obj_name, api.parse_autotaskprocessexception(e))
 
-                error_messages += '{}\n'.format(msg)
-                failed_classes += 1
+            except AutotaskAPIException as e:
+                error_msg = ERROR_MESSAGE_TEMPLATE.format(
+                    obj_name, api.parse_autotaskapiexception(e))
 
             except SAXParseException as e:
-                msg = 'Failed to connect to Autotask API. ' \
+                error_msg = 'Failed to connect to Autotask API. ' \
                       'The error was: {}'.format(e)
 
-                error_messages += '{}\n'.format(msg)
-                failed_classes += 1
+            finally:
+                if error_msg:
+                    self.stderr.write(error_msg)
```

Comment:
I think this is unnecessary since the error message gets displayed when the `CommandError` is raised at line 141.

Reply:
I added it so it works the same as cwsync, rather than having both these methods work and report in different ways.

Reply:
Ok, no problem. It just bothered me that in duplicated error messages after a failed sync. Not a big deal though. I'll just run through some final testing and then merge.